python/meep.i

```diff
@@ -284,6 +285,16 @@ PyObject *py_do_harminv(PyObject *vals, double dt, double f_min, double f_max, i
   delete $1;
 }
 
+%typecheck(SWIG_TYPECHECK_POINTER) material_type _default_material {
```

Comment:
Shouldn't this typecheck etc. apply to any `material_type` parameter, not limited to those named `_default_material`?

Reply:
Fixed.